include/class.queue.php

```diff
@@ -2825,7 +2826,8 @@ class DateTimeFilter
     static $desc = /* @trans */ "Date and Time";
 
     function filter($text, $row) {
-        return $text->changeTo(Format::datetime($text->value));
+        return $text ?
```

Comment:
Should this be $text->value instead of just $text?

Reply:
No text can be null, causing a fatal error.